### ISSUE TITLE
Update NDBC for Pandas

### ISSUE THREAD

**Issue by dopplershift** [Unidata/siphon#265]
Apparently:

>/Users/rmay/miniconda3/envs/unidata/lib/python3.7/site-packages/siphon/simplewebservice/ndbc.py:107: FutureWarning: read_table is deprecated, use read_csv instead.

Not sure how that will work, but here are the current offenders:
```
siphon/simplewebservice/ndbc.py
106:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
144:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
178:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
226:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
265:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
295:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
323:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
360:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
394:        df = pd.read_table(StringIO(content), comment='#', na_values='MM',
454:        df = pd.read_table(StringIO(resp.text), comment='#', na_values='MM',
```

**Comment by zbruick** [comment]
`read_table` has been changed to `read_csv` in #269. All tests pass without any other changes needed, so I think this can be closed once the PR is merged.

**Comment by zbruick** [comment]
Closed by #269.
